docker-hub/publish/insights-analytics.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Insights & analytics
+description: Provides an API analytics tool for your products.
+keywords: docker hub, hub, insights, analytics, api
+---
+
+The Insights & Analytics tool includes an API that provides analytics for your products. With this API, you now have self-serve access to metrics where you can download both raw data and summary data. You can view how many times your images have been pulled by tag or by digest, and get breakouts by geolocation, cloud provider, and tool.The API measures the number of pulls that were triggered by users, and automated machines such as CI/CD systems. You can import this weekly or monthly data into your own systems or you can analyze it manually using spreadsheet reports. The data can then be exported as a downloadable GZipped CSV file. Review the Data Definitions section for more information.
+
+There are automated systems that frequently check for new versions of your images, and we have tried to distinguish between those 'version checks' and real user downloads by inspecting the order and timing of API calls from the same IP address. This will provide more insight into customer behavior. You can inspect the rules we have applied in the Action Classification Rule Set section of this document. We welcome [feedback](https://forms.gle/nb7beTUQz9wzXy1b6){: target="_blank" rel="noopener" class="_"} and questions on these rules.
+
+## Raw data
+
+In your raw data CSV you will have access to the data points listed below. You can request raw data for a complete week (Monday through Sunday) or for a complete month (available on the first day of the following month). **Note:** each action is represented as a single row.
+
+  * Type (industry)
+  * Host (cloud provider)
+  * Country (geolocation)
+  * Timestamp
+  * Namespace
+  * Repo
+  * Reference (digest is always included, tag is provided when available)
+  * HTTP method
+  * Action, one of the following:
+    * Pull by tag
+    * Pull by digest
+    * Version check
+  * User-Agent Tool
+
+## Summary data
+
+In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday through Sunday) or for a complete month (available on the first day of the following month).
+
+For every namespace, repository, and reference (tag or digest):
+
+  * Unique IP address count
+  * Pulls by tag count
+  * Pulls by digest count
+  * Version check count
+
```

Comment:
We should add a section here about accessing the data via the UI. 

- where to access the UI (example: https://hub.docker.com/orgs/docker/insights)
- who has access to the UI and API (and team member of an organization)
- add screenshot of the UI

Reply:
Hey Britney. I added a new section for the UI. Please check it out! And for who has access, I'm not sure if I got it right but I added this to the first paragraph:

All members of an organization has access to the analytics data.